### DISCUSSION TITLE
creating a listing

### DISCUSSION THREAD

**Discussion by shiyuleixia** [ProjectOpenSea/seaport#547] [Q&A] [answered]
I want to create a listing with the seaport version two api:v2/orders/ethereum/seaport/listings   and I have composed the order_parameters parameter:
{
  "offerer": "0x88edCa617B8E4fF3793d06F42E5b80225c2f5AdA",
  "zone": "0x00000000E88FE2628EbC5DA81d2b3CeaD633E89e",
  "zone_hash": "0x0000000000000000000000000000000000000000000000000000000000000000",
  "start_time": "1655458073",
  "end_time": "1658050073",
  "order_type": 0,
  "salt": "123456",
  "conduitKey": "0x0000007b02230091a7ed01230072f7006a004d60a8d4e71d599b8104250f0000",
  "nonce": "0",
  "offer": [
    {
      "item_type": 3,
      "token": "0x7f04504AE8DB0689A0526d99074149FE6dDf838c",
      "identifier_or_criteria": "10",
      "startAmount": 1,
      "endAmount": 1
    }
  ],
  "consideration": [
    {
      "item_type": 0,
      "token": "",
      "identifier_or_criteria": "",
      "startAmount": 0.5,
      "endAmount": 0.5,
      "recipient": "0x1f5e0376def810b7a46d72bfb561460bcd686288"
    }
  ]
}

there are two question with  me :
first.  where I could get or create the signature which this api need
sencode. I just leave the signature blank ,then I push the try button,but the request was always on loading state：
![image](https://user-images.githubusercontent.com/15072820/174290705-8fd5db9c-373b-4ec9-8001-be20c1873099.png)

**Comment by pamungkaski** [comment]
Wanted to help answering this, so the signature are created by asking the user to sign the payload data of the listings. The signature use EIP 712 typed data signing. Check https://github.com/ProjectOpenSea/seaport-js

**Comment by jack75015** [comment]
Hey
I tried the same way you did but got:
```
{
  "errors": [
    "Chain with identifier ethereum does not exist in the db"
  ]
}
```

**Comment by njovy** [comment, marked as answer]
@jack75015 Unfortunately seaport doc contains multiple errors. This was the first error I saw when I tried to create a new order.  However an error message returned from the server is pretty straight forward to debug. You have to change the endpoint to ethereum to rinkeby. https://testnets-api.opensea.io/v2/orders/rinkeby/seaport/listings is the right url for a testnet.

**Comment by jack75015** [comment]
> @jack75015 Unfortunately seaport doc contains multiple errors. This was the first error I saw when I tried to create a new order. However an error message returned from the server is pretty straight forward to debug. You have to change the endpoint to ethereum to rinkeby. https://testnets-api.opensea.io/v2/orders/rinkeby/seaport/listings is the right url for a testnet.

Yeah @njovy thanks
Yeah, I just see it after some hours :)
The docs should really be updated :)

**Comment by Slokh** [comment]
Thanks guys, I've updated the API reference. General docs overhaul will be in the works for Q3

**Reply by 0xAskar** [reply]
I'm glad yall are working on it, but my only grievance with this, is if you're going to shift your api usage and platform to a new system with poor documentation, then you render useless many current platforms depending on those changes. So, I don't understand how the documentation isn't as thorough and planned out _before the release_.
